diagnostic.md

```diff
@@ -6,70 +6,81 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+you can look at the your scripts examples or go into your migrate folder
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+you need need to go into routes.rb and controllers
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+So bundle exec rake db:migrate executes the rake script with the command db:migrate
+in the context of the current bundle.
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+
+The Scaffold name needds to be:
+lower_snake_case
+SINGULAR
+
+The controller name needs to be:
+UpperCamelCase
+PLURALIZED
+
```

Comment:
The first command generates routes, a controller and model for foo. The second generates just a foo controller.